app/graphql/mutations/create_in_progress_submission_report.rb

```diff
@@ -0,0 +1,46 @@
+module Mutations
+  class CreateInProgressSubmissionReport < ApplicationQuery
+    include QueryAuthorizeCoach
+    include ValidateSubmissionGradable
+
+    argument :submission_id, ID, required: true
+    argument :description, String, required: true
```

Comment:
Do either of _queued_ or _in-progress_ submission reports need a description? 🤔

The description field is meant to hold the Markdown text of the report, right?

If a description is required for either of these states, both the related mutations also require length-limit checks.

Reply:
Not fully sure. Keeping it as an optional field. Foreseeing some useful custom information we can add in our workflows. The UI already handles this anyway.